### PR TITLE
10 postgis import data of buildings needed to teach model

### LINKED ISSUE #10: Postgis: Import data of buildings needed to teach model

### PR DESCRIPTION
**Pull request by JakubCzyzewski-TomTom**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -12,5 +12,5 @@ PREDICTION_LOCAL_PORT=5000
 PREDICTION_DOCKER_PORT=5000
 
 # for local testing change to localhost, for docker to host.docker.internal
-# POSTGRES_URL=jdbc:postgresql://host.docker.internal:${POSTGRES_LOCAL_PORT}/${POSTGRES_DATABASE}
-POSTGRES_URL=jdbc:postgresql://localhost:${POSTGRES_LOCAL_PORT}/${POSTGRES_DATABASE}
\ No newline at end of file
+POSTGRES_URL=jdbc:postgresql://host.docker.internal:${POSTGRES_LOCAL_PORT}/${POSTGRES_DATABASE}
```
Comment (RadekBucki) [inline .env]: I think that probably this cause fail of github workflow

File: src/main/java/pl/ioad/adoto/database/DBService.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package pl.ioad.adoto.database;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import pl.ioad.adoto.database.dto.BuildingDTO;
+import pl.ioad.adoto.database.dto.GeometryMapper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@org.springframework.stereotype.Service
```
Comment (RadekBucki) [inline src/main/java/pl/ioad/adoto/database/DBService.java]: Please add this to import

File: src/main/java/pl/ioad/adoto/database/dto/BuildingDTO.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class BuildingDTO {
+    Integer id;
+    GeometryDTO geometry;
```
Comment (RadekBucki) [inline src/main/java/pl/ioad/adoto/database/dto/BuildingDTO.java]: Missed access qualifiers

File: src/main/java/pl/ioad/adoto/database/dto/GeometryDTO.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class GeometryDTO {
+    Double[][] coordinates;
+    String type;
```
Comment (RadekBucki) [inline src/main/java/pl/ioad/adoto/database/dto/GeometryDTO.java]: Missed access qualifiers

**Review by RadekBucki — CHANGES_REQUESTED** [review]

**Review by RadekBucki — APPROVED** [review]

**Review by Damian2057 — APPROVED** [review]

**Pull request merged**